xml/admin_monitoring_alerting.xml

```diff
@@ -0,0 +1,936 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"  xml:id="monitoring-alerting">
+ <title>Monitoring and Alerting</title>
+ <para>
+  By default, &deepsea; deploys a monitoring and alerting stack on the
+  &smaster;, allowing for multiple instances for a HA setup.
+  It consists of the following components:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    <emphasis role="bold">&prometheus;</emphasis> monitoring and alerting
+    toolkit.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    <emphasis role="bold">&grafana;</emphasis> visualization and alerting
+    software.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The <systemitem class="daemon">prometheus-node_exporter</systemitem>
+    service running on all &sminion;s.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  The &prometheus; configuration and <emphasis>scrape</emphasis> targets
+  (exporting daemons) are setup automatically by &deepsea;. &deepsea; also
+  deploys a list of default alerts, for example <literal>health
+  error</literal>, <literal>10% OSDs down</literal>, or <literal>pgs
+  inactive</literal>.
```

Comment:
Add a section here with possible custom configuration as outlined here https://github.com/SUSE/DeepSea/wiki/New-Monitoring

Reply:
Are you referring to: https://github.com/SUSE/DeepSea/wiki/New-Monitoring#pillar-variables ?

Reply:
spot on :+1: